core/src/test/java/com/linecorp/armeria/server/ServerBuilderTest.java

```diff
@@ -641,4 +642,46 @@ void defaultTcpUserTimeoutApplied() {
                 .containsExactly(entry(TCP_USER_TIMEOUT, userDefinedValue),
                                  entry(SO_LINGER, lingerMillis));
     }
+
+    @Test
+    void exceptionReportInterval() {
```

Comment:
But perhaps there's no need to rename the test methods in this class

Reply:
Sure ~